pkg/mcs/resourcemanager/server/metering.go

```diff
@@ -27,15 +27,16 @@ const (
 	ResourceManagerCategory = "resource-manager"
 	ruMeteringVersion       = "1"
 
-	meteringDataOltpRUField              = "oltp_ru"
-	meteringDataOlapRUField              = "olap_ru"
+	meteringDataOLTPRUField              = "oltp_ru"
+	meteringDataOLAPRUField              = "olap_ru"
+	meteringDataWriteBytesField          = "write_bytes"
```

Comment:
Only the write bytes need to be considered?

Reply:
Yes, currently we only need to add this field after confirming with PM.